_redirects

```diff
@@ -0,0 +1 @@
+/ https://instantsearchjs.netlify.app/specs/
```

Comment:
other pages also need to be redirected, I think it should be:

```suggestion
/* https://instantsearchjs.netlify.app/specs/:splat !
```

We also need to change the netlify build to do nothing instead of building the site

Reply:
Should we do this after it builds redirects one time?

Reply:
yes, this setup only redirects the root, not other pages like https://instantsearch-css.netlify.app/widgets/numeric-menu/